client/src/components/FeedCards.js

```diff
@@ -33,14 +33,11 @@ class FeedCards extends Component {
                     if (error) {
                        return <div>{error.message}</div>;
                     } else if (props) {
-                        let cards = []
-                        for (let i = 0; i < props.user.length; i++) {
-                            if (props.user[i].visible && props.user[i].uuid !== this.props.user_id) {
-                                cards.push(<UserCard name={props.user[i].name} grad_year={props.user[i].grad_year} school={props.user[i].school} contact={props.user[i].contact} skills={props.user[i].skills.filter(function (el) {
-                                    return Boolean(el);
-                                })} experience={props.user[i].experience} />);
-                            }
-                        }
+                        let cards = props.user.map(user => {
```

Comment:
```suggestion
                        const cards = props.user.map(user => {
```

looks like you can use const here